File: src/typescript/check/function/getUserName.ts
function getUserName(userId: number): { name: string | null; error: string | null } {
    if (userId === 1) {
        return { name: "Alice", error: null };
    } else {
        return { name: null, error: "User not found." };
    }
}

const result = getUserName(2);
console.log("User Name:", result.name!.toUpperCase());
\ No newline at end of file

File: truth_set/python/check/function/getUserName.py
def get_user_name(user_id):
    if user_id == 1:
        return {"name": "Alice", "error": None}
    else:
        return {"name": None, "error": "User not found."}

result = get_user_name(2)
print("User Name:", result["name"].upper())